docs/custom-vnet.md

```diff
@@ -0,0 +1,170 @@
+# Using a custom virtual network with Azure Container Service 
+In this tutorial you are going to learn how to use [ACS Engine](https://github.com/Azure/acs-engine) to deploy a brand new cluster into an existing or pre-created virtual network. 
+By doing this, you will be able to control the properties of the virtual network or integrate a new cluster into your existing infrastructure.
+
+*Note: This article describes the procedure with Docker Swarm but it will work in the exact same way with the all the orchestrators available with ACS Engine: Docker Swarm, Kubernetes and DC/OS.*
```

Comment:
Put a note here that for Kubernetes, it must be deployed in same RG, and permissions must be set according to the kubernetes walkthrough.